pyproject.toml

```diff
@@ -1,3 +1,76 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "trio"
+description = "A friendly Python library for async concurrency and I/O"
+authors = [{name = "Nathaniel J. Smith", email = "njs@pobox.com"}]
+license = {text = "MIT OR Apache-2.0"}
+keywords = [
+    "async",
+    "io",
+    "networking",
+    "trio",
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
```

Comment:
Is this still correct?

Reply:
definitely worth fixing the README.rst and this at once in a separate PR. I want to keep the metadata identical in the PR

Reply:
where's the issue to switch from alpha 3 to final? and be more confident in introducing trio in the docs?